src/pages/dashboardGroupEdit/components/GroupForm.tsx

```diff
@@ -150,12 +150,16 @@ export default function GroupForm({
     (() =>
       history.push(
         resolvePathVariables(
-          isEdit
+          isEdit || isClone
             ? DASHBOARD_GROUPS_ROUTES.PARTY_GROUPS.subRoutes.PARTY_GROUP_DETAIL.path
             : DASHBOARD_GROUPS_ROUTES.PARTY_GROUPS.subRoutes.MAIN.path,
           {
             partyId: party.partyId,
-            groupId: (initialFormData as PartyGroupOnEdit).id,
+            groupId: isEdit
+              ? (initialFormData as PartyGroupOnEdit).id
+              : isClone && partyGroupCloneId
+              ? partyGroupCloneId
```

Comment:
partyGroupCloneId is the id of the group cloned, not the id of the new group.
Actually we don't know the id of the new group, thus we cannot complete the issue

This behavior is not aligned with the creation of groups, which redirect the user to the list of groups